packages/app/src/cli/services/dev/ui/components/Dev.tsx

```diff
@@ -1,7 +1,7 @@
 import metadata from '../../../../metadata.js'
 import {DeveloperPlatformClient} from '../../../../utilities/developer-platform-client.js'
 import {ExtensionInstance} from '../../../../models/extensions/extension-instance.js'
-import {devSessionStatus} from '../../processes/dev-session.js'
+import {devSessionStatusManager, DevSessionStatus} from '../../processes/dev-session-status-manager.js'
```

Comment:
I feel like `devSessionStatusManager` should be a prop for the component, then we can get rid of the global state? If we create it we can pass it into `dev-session` / `Dev`.

Reply:
Yeah, i think that makes sense, I was going to do that as part of this PR, but it would imply MANY  changes (specially in tests) to pass it down to both processes.

Is it ok if I do that in a new PR in the stack? I think it'd be cleaner